.github/workflows/test.yaml

```diff
@@ -0,0 +1,143 @@
+name: CI PUSH PR ANALYZE AND TEST
+
```

Comment:
Could you please please rename this workflow to use standard case (not upper)?

Reply:
Done :)